src/parse.ts

```diff
@@ -1,4 +1,4 @@
-import { URL } from 'url'
+import { URL } from './universal-url'
```

Comment:
I think you can just use URL since it's global in browser (via window) and Node.js (via globalThis), so we can't drop the typecheck code of `src/universal-url.ts`, saving some bytes 🙂

Reply:
Hey, unfortunately, if you import the URL, the web app will break. If you want to try it out [here](https://codesandbox.io/p/sandbox/url-test-hv23fj?file=%2Fsrc%2FApp.js%3A12%2C1). 

![CleanShot 2024-05-23 at 04 56 02@2x](https://github.com/TooTallNate/spotify-uri/assets/37520667/032c8f24-189e-4eb8-be2f-5d865a5d8777)

Reply:
What Kiko means is that this `universal-url` file isn't necessary at all. The global `URL` class is available in all modern JS environments.

Reply:
I'm fine with dropping older (EOL) Node.js version support.